common/config/rush/common-versions.json

```diff
@@ -45,7 +45,7 @@
     // for it here and it should be removed once core-paging 1.2.0 is released.
     "@azure/core-paging": [
       "^1.2.0",
-      "^1.2.0-alpha"
+      ">=1.2.0-alpha <1.2.0-alphb"
```

Comment:
Do we need both `^1.2.0-alpha` and `>=1.2.0-alpha <1.2.0-alphb`?  The previous error specifically wanted `^1.2.0-alpha`.

Reply:
Completed TA pipeline with dev version successfully with this change here. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1052208&view=results